_tests/browser_tests/page_objects/page_the-bureau.js

```diff
@@ -0,0 +1,10 @@
+var TheBureauPage = function() {
+  this.get = function() {
+    browser.get( '/the-bureau' );
```

Comment:
Trivial: some tests include the trailing slash for the URL, others don't. Might make that consistent.